dashboard/src/shared/utils.ts

```diff
@@ -153,3 +181,34 @@ export function getAppStatusLabel(
   const jsonReason = installedPackageStatus_StatusReasonToJSON(statusReason);
   return jsonReason.replace("STATUS_REASON_", "").toLowerCase();
 }
+
+export function getSupportedAuthMethods(
```

Comment:
```suggestion
export function getSupportedPackageRepositoryAuthTypes(
```
Just being a nitpicker here 😄